GeoDjango
controlling
don
package
Property
gdal
of
place
objects
Default
preventing
isn
ImportError
gis
available
bad
_ptr
__class__
c_void_p
isinstance
pointer
dummy
outside
Base
A
C
django
encountered
NULL
numpy
supported
want
_set_ptr
HAS_GDAL
module
may
a
Pointer
access
geos
t
s
NumPy
library
contrib
to
libraries
allowed
raises
location
exception
since
used
type
has
invalid
ptr_type
when
underlying
ctypes
be
GDALInfo
Incompatible
Raise
property
GEOSException
Only
GEOSBase
that
TypeError
GDAL
Have
Trying
this
passing
ptr
valid
Initially
allow
pointers
set
object
routines
_get_ptr
controls
GEOJSON
NoneType
compatible
types
GEOS
attempting
error
thus
__name__
the
memory
Using
an
very
